project/ISSUES.md

```diff
@@ -0,0 +1,171 @@
+common problems we've tried to address in this approach are:
+
+* The issue graveyard -- where, once off the first 'page' of issues or below
+  the 'fold' in Trello, things sometimes do not resurface and are lost to
+  history.
+
+* Difficulty clearly presenting and maintaining a high level project roadmap
+  and associated meta-issues.
+
+
+Milestones
+==========
+
+Every pull request should, prior to merge, be assigned to the milestone
+corresponding to the Galaxy release it will first appear in (15.10, 16.01, and
+so on).  This, along with the tags applied, will be used to generate the
+release notes.
+
+Any non-PR issue assigned to a milestone *must* be resolved or reassigned prior
+to publishing that release -- that is, this is the new way to create and track
+"Issues Blocking Release".  This will be a primary way to report bugs and force
+them to be reconciled (fixed, closed, or intentionally postponed) prior to
+release.  In practice, bugs should almost always be tagged with a milestone
+which forces the reconciliation date.  Issues *may* be, but they don't
+necessarily have to be -- this is subjective and it depends on if the submitter
+(or any other contributor) wants to force the issue to be revisited at the next
+release.
+
+Effective use of milestones should prevent bugs from falling through the
+cracks, and will provide a mechanism for forcing the revisitation (and thus
+progress or even potential discard) of ideas for enhancements or features.
+
+
+Labeling Structure
+==================
+
+To allow for easy search, filtering, and general issue management every issue
+or PR (not tagged `Procedures` or `Planning`) is *required* to have three
+labels which indicate the type, status, and focus area of the issue.  Any issue
+without these three tags will be automatically have a `triage` label applied
+indicating that it needs human intervention to be correctly tagged.  These
+`triage` tagged issues will be regularly reviewed and tagged as appropriate.
+
+Type Labels
+-----------
+
+The 'class' label set is used for classifying the type of contribution or
+request/report to separate enhancements and new features from bugs, etc.
+
+* `class/bug` - something is broken, and it needs fixing
+* `class/documentation` - documentation is unclear or can be improved
+* `class/enhancement` - polish to an existing feature or interface
+* `class/feature` - something brand new
+
+Status Labels
+-------------
+
+The `status` of an issue should be tracked using the following stages:
+
+* `status/triage` - brand new issue/pr that doesn't offer a concrete plan or
+  solution
+* `status/planning` - issue reviewed and has a sufficiently detailed primary
+  message and/or commentary
+* `status/WIP` - this issue or PR is currently being worked on.  It should not
+  be merged (or closed) without pinging the owner/submitter.
+* `status/review` - issue is resolved or PR is complete and needs review
```

Comment:
Yeah, my intent was that it indicated the issue needed some extra verification prior to close.  That might not be a useful indicator for issues, though, and I'd expect many issues to go from WIP to closed (especially when using the GitHub auto-close like "resolves #ISSUENUMBER".